build-system/tasks/css/jsify-css.js

```diff
@@ -99,13 +100,37 @@ async function transformCssString(contents, opt_filename) {
  */
 async function jsifyCssAsync(filename) {
   const {contents, hash: filehash} = await batchedRead(filename);
-  const hash = md5(filehash, await getEnvironmentHash());
+  const imports = await getCssImports(filename);
+  const importHashes = await Promise.all(
+    imports.map(async (importedFile) => (await batchedRead(importedFile)).hash)
+  );
+  const hash = md5(filehash, ...importHashes, await getEnvironmentHash());
   const result = await transformCss(contents, hash, filename);
 
   result.warnings.forEach((warn) => log(red(warn)));
   return result.css + '\n/*# sourceURL=/' + filename + '*/';
 }
 
+/**
+ * Computes the transitive closure of CSS files imported by the given file.
+ * @param {string} cssFile
+ * @return {Promise<!Array<string>>}
+ */
+async function getCssImports(cssFile) {
+  const contents = await fs.readFile(cssFile);
+  const topLevelImports = cssImports(contents)
+    .map((result) => result.path)
+    .filter((importedFile) => !importedFile.startsWith('http'))
+    .map((importedFile) => path.join(path.dirname(cssFile), importedFile));
+  if (topLevelImports.length == 0) {
+    return topLevelImports;
+  }
+  const nestedImports = await Promise.all(
+    topLevelImports.map(async (file) => await getCssImports(file))
```

Comment:
nit: the async/await is unnecessary for the arrow function

Reply:
Good point. I included the `async/await` on line 129 to be consistent with line 105, where the `await` is necessary in order to extract `.hash`.

https://github.com/ampproject/amphtml/blob/5666379e3fb1af0215222103942cce36b9203281/build-system/tasks/css/jsify-css.js#L105

If there's shorthand that's applicable in both places, happy to send a follow up. Otherwise will leave this as is until a future refactor of CSS code.